.github/workflows/build-syncthing.yaml

```
@@ -337,6 +338,31 @@ jobs:
             *.tar.gz
             compat.json
 
+  package-illumos:
+    runs-on: ubuntu-latest
+    name: Package for illumos
+    steps:
+      - uses: actions/checkout@v4
```

v5 was [released](https://github.com/actions/checkout/releases) 3 weeks ago.

```suggestion
      - uses: actions/checkout@v5
```

---

We have v4 everywhere else, so I think this is fine until we bump it globally.